docs/keepalive.md

```diff
@@ -0,0 +1,50 @@
+# Keepalive
+
+gRPC sends HTTP2 pings on the transport to detect if the connection is down.
+If the ping is not acknowledged by the other side within a certain period, the connection
+will be closed. Note that pings are only necessary when there is no activity on the connection. 
+
+## What should I set?
```

Comment:
This reads as if we're suggesting these value are used instead of the defaults. I think it might be clearer to show the default values first and then follow with how users can configure them (i.e. providing `{Client,Server}ConnectionKeepalive` to the relevant builder).

Reply:
I'm not sure my latest commit satisfies your comment. Feel free to update this doc on your own.